subprojects/workers/src/main/java/org/gradle/workers/internal/DaemonForkOptionsBuilder.java

```diff
@@ -63,7 +55,7 @@ public DaemonForkOptionsBuilder withClassLoaderStrucuture(ClassLoaderStructure c
     }
```

Comment:
nitpick: there's a typo in the method name (struc*u*ture)

Reply:
Fixed